BigQuery/14_final_schema.sql

```diff
@@ -0,0 +1,101 @@
+
+-- This schema is *SIMILAR* but not exactly the same as the PNC 2019 Challenge Schema
+-- Changing this to the correct schema should be very simple
+-- Please feel free to modify this yourself (add/rename/remove fields) to get it into the schema needed
+
+CREATE OR REPLACE TABLE `physionet.final_schema` as
+
+SELECT 
+subject_id,
+hadm_id,
+stay_id,
+chart_hour,
+admittime,
+deathtime,
+admission_type,
+first_careunit,
+last_careunit,
+total_los,
+ethnicity,
+gender,
+anchor_age as age,
+-- Cultures fields
+blood_cx,
+urine_cx,
+sputum_cx,
+-- Antibiotics fields (input events)
+amount as antibiotics_amount,
+rate as antibiotics_rate,
+--Antibiotics fields (pharmacy)
+abx_orders,
+-- vitals
+resp_rate,
+heart_rate,
+temperature,
+systolic_bp,
+diasystolic_bp as diastolic_bp,
+mean_arterial_bp,
+etco2,
+spo2,
+-- labs
+bicarbonate,
+chloride,
+bilirubin_indirect,
+bilirubin_direct,
+bilirubin_total,
+calcium,
+creatinine,
+glucose,
+lactate,
+magnesium,
+phosphate,
+potassium,
+troponin_I,
+hematocrit,
+hemoglobin,
+ptt,
+wbc,
+fibrinogen,
+platelets,
+-- ABG
+PaO2,
```

Comment:
I left this out of other ones, but I feel strongly about capitalization consistency for column names. The reason is that if we're using lowercase for some but not others, it's going to be confusing to pull from. This pushes logic for creating the end datasets to look like the PNC or whatever to somewhere else, but the table definitions should be pristine. What do you think?

Reply:
I think sticking to lowercase makes sense - I had always seen "PaO2" as (Pa)rtial pressure of (O2)xgen and wasn't sure if a flat pao2 would be misread as something else, like "pao 2", which is silly in hindsight now that I'm more familiar with what kinds of fields are availble in EHR data

Reply:
👍